.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: generic
+services:
+  - docker
+
+notifications:
+  email:
+    on_success: change
+    on_failure: always
+env:
+  global:
+    - ROS_REPO=ros
+    - NOT_TEST_INSTALL=true
+    - VERBOSE_TESTS=false
+    - UPSTREAM_WORKSPACE=file
+  matrix:
+    - ROS_DISTRO=kinetic
+install:
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
```

Comment:
this is not the latest default `.travis.yml` we use - where did you get that from?

Reply:
use https://github.com/mojin-robotics/mojin_drivers/blob/kinetic-devel/.travis.yml as an example

Reply:
I gave it another try